### ISSUE TITLE
Messages logged once per user

### ISSUE THREAD

**Issue by Bastrabun** [acmgit/smart_chat#4]
The loop starting in https://github.com/acmgit/smart_chat/blob/master/lib.lua#L292 leads to a debug message printed out once for each online user. I suggest raising the log level to DEBUG for this statement.

**Comment by acmgit** [comment]
Thanks, i changed the level to debug, because it is not necessary to print out for each online user.

**Comment by Bastrabun** [comment]
2022-04-21 02:07:17: WARNING[Server]: Tried to log at unknown level 'debug'.  Defaulting to "none".

Seems like https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L4795 allows only

`"none"`, `"error"`, `"warning"`, `"action"`, `"info"`, or `"verbose"`.  Default is `"none"`.

No clue how I got the impression `"debug"` was a valid loglevel. Maybe due to `minetest.debug(...)` : Equivalent to `minetest.log(table.concat({...}, "\t"))`
